lib/ZnapZend.pm

```diff
@@ -520,15 +521,31 @@ my $daemonize = sub {
     }
 };
 
+my $keepalive = sub {
+    my $self = shift;
+
+    $self->zLog->debug('Keepalive started.');
+
+    # workaround for https://github.com/oetiker/znapzend/issues/168
+    my $heartbeat = Mojo::IOLoop->recurring(1 => sub { my $loop = shift; select(undef, undef, undef, 0.9); });
+
```

Comment:
this is actually pretty evil ... we are in an event loop here and you are sleeping away for 0.9 seconds every second ... the whole point of writing event based programs is that all your tasks should return control to the event loop as quickly as possible ...sleeping is sort of the ultimate mockery ...